packages/vouching/package.json

```diff
@@ -13,7 +13,9 @@
   "author": "",
   "license": "MIT",
   "dependencies": {
-    "openzeppelin-solidity": "^1.12.0"
+    "openzeppelin-solidity": "^1.12.0",
```

Comment:
We should aim at removing openzeppelin-solidity altogether later down the road, if we're using -zos

Reply:
sounds good, we were only using it "locally" on `Vouching`, removing it, but we still need it throughout in TPL.. will wait until we encapsulate TPL to remove the dep here.